### ISSUE TITLE
3 security vendors flagged this file as malicious

### ISSUE THREAD

**Issue by H4xl0r** [AutomationSoftwareInc/eBay_Auto_Bid_Sniper_Software#3]
https://www.virustotal.com/gui/file/af5efb28a44c6cbf5b1399f3a46bff7f05f2c74d32c91d0ee3f3df718313c752

**Comment by AutomationSoftwareInc** [comment]
> https://www.virustotal.com/gui/file/af5efb28a44c6cbf5b1399f3a46bff7f05f2c74d32c91d0ee3f3df718313c752

No, don't worry that report is false you can safely ignore it, it's pretty common to RECIEVE false reports from anti virus softwares whenever running any kind software, try running it thru a more reputable virus scanner like mcafee or Norton and you will get accurate results!!!
